merlin-sdk/src/main/java/gov/nasa/jpl/aerie/merlin/protocol/types/SerializedValue.java

```diff
@@ -73,6 +116,11 @@ public <T> T match(final Visitor<T> visitor) {
       return visitor.onNumeric(value);
     }
 
+    @Override
```

Comment:
These getValue() implementations could be private, but it seems like a convenient method to have.  I suspect there's a good reason why SerializedValue is not SerializedValue<T>.  I guess it might be a breaking change at this point.

Reply:
Yeah, SerializedValue is intentionally not generic because it is intended to be basically a JSON object, which is type checked at runtime. Its always used in a context that is polymorphic over all types, so if we parameterized it, it would just be used as `SerializedValue<Object>` all the time.